### ISSUE TITLE
AccessibilityManager: Cannot read property 'removeEventListener' of undefined

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#94760]
OS X, zshell, turn on VoiceOver, have terminal opened
Notice stack trace in the dev tools console:
Sorry for not very clear steps, if the stack trace is not enough let me know and I can try to find exact steps. Thanks

```
TypeError: Cannot read property 'removeEventListener' of undefined
    at t._onResize (AccessibilityManager.ts:175)
    at t._refreshRowsDimensions (AccessibilityManager.ts:286)
    at AccessibilityManager.ts:105```

[93697:0409/102344.257301:INFO:CONSOLE(212)] "TypeError: Cannot read property 'removeEventListener' of undefined
    at t._onResize (/Users/isidor/Development/vscode/node_modules/xterm/lib/xterm.js:1:204596)
    at t._refreshRowsDimensions (/Users/isidor/Development/vscode/node_modules/xterm/lib/xterm.js:1:206714)
    at /Users/isidor/Development/vscode/node_modules/xterm/lib/xterm.js:1:203244", source: /Users/isidor/Development/vscode/out/bootstrap-window.js (212)
```
